### PR TITLE
Qz test

### PR DESCRIPTION
**Pull request by qzhang234**
- Added customized Rigaku detector Ophyd class (`rigaku500k`) in `/device`;

- Added customized Rigaku ZDT acquisition in `/plan`;  
   - The intention is to have dedicated plan for each main operation mode

- Added metadata device for user directories/qmap (`Init_User`) and sample info (`Init_QNW`) in `/device`.

### CODE REVIEW

**Comment by qzhang234** [comment]
@prjemian  Oh no that was a mistake, it was supposed to be under `~/bluesky_data`. I was actually looking for that folder `2021`, so glad you found it.

It's now moved out of the repo and I just did another push

File: profile_bluesky/startup/instrument/devices/metadata_carrier.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+]
+
+
+from ophyd import Device
+from ophyd import EpicsSignal
+from ophyd import Component as Cpt
+from bluesky import plan_stubs as bps
+
+
+class Init_User(Device):
```
Comment (prjemian) [inline profile_bluesky/startup/instrument/devices/metadata_carrier.py:16]: change to `BasicUserInformation`

File: profile_bluesky/startup/instrument/devices/metadata_carrier.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+
+    def select_qmap(self, qmap_name: str):
+        """
+        set qmap
+        """
+        yield from bps.mv(self.qmapname, f'/home/8-id-i/{aps.aps_cycle.get()}/{qmap_name}')
+
+
+class Init_QNW(Device):
```
Comment (prjemian) [inline profile_bluesky/startup/instrument/devices/metadata_carrier.py:40]: *refactor* to `QnwParameters`

**Review by prjemian — APPROVED** [review]
LGTM

**Pull request merged**